forc-plugins/forc-client/src/op/deploy.rs

```diff
@@ -64,7 +84,7 @@ pub async fn deploy_pkg(
     let client = FuelClient::new(node_url)?;
 
     let bytecode = compiled.bytecode.clone().into();
-    let salt = Salt::new([0; 32]);
+    let salt = command.salt.salt.unwrap_or_default();
```

Comment:
Would it make sense to support the ability to generate a random salt for the user? Would make things like QuickStart doc examples easier if users could just copy paste commands without needing to customize arguments etc

Reply:
Yeah that sounds useful :+1: I'll add a `--random-salt` flag now unless there's any objections?